assets/javascripts/kitten/components/typography/title/stories.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import {
+  withKnobs,
+  text,
+  boolean,
+  select,
+} from '@storybook/addon-knobs'
+import { Title } from './index'
+import { Marger } from '../../layout/marger'
+import { Container } from '../../grid/container'
+
+const modifierOptions = {
+  Primary: 'primary',
+  Secondary: 'secondary',
+  Tertiary: 'tertiary',
+  Quaternary: 'quaternary',
+  Quinary: 'quinary',
+  Senary: 'senary',
+  Septenary: 'septenary',
+}
+
+export default {
+  component: Title,
+  title: 'Typography/Title',
+  decorators: [withKnobs],
+
+  parameters: {
+    component: Title,
+    componentSubtitle: 'List of Title',
```

Comment:
ça correspond à quoi le sous-titre "List of Title" ? je vois pas de liste dans la story

Reply:
Oui t'as raison je vais l'étoffer !